.env.example

```diff
@@ -173,3 +173,6 @@ PERIODICAL=stop_expired_deploys:60,remove_expired_locks:60,report_system_stats:6
 # GCLOUD_OPTIONS=--verbose    # optional, options
 # GCLOUD_PROJECT=foo-123
 # GCLOUD_ACCOUNT=my-account
+
+## Feature: Only admins can (un)lock stages which affect production.
+# FEATURE_ONLY_ADMINS_CAN_LOCK_UNLOCK_PRODUCTION_STAGE=1
```

Comment:
`ONLY_ADMINS_CAN_LOCK_UNLOCK_PRODUCTION_STAGE` should be long enough :D

Reply:
... maybe `PRODUCTION_STAGE_LOCK_REQUIRES_ADMIN`

Reply:
yeah, naming things is always hard ;-)  I like PRODUCTION_STAGE_LOCK_REQUIRES_ADMIN.